source/standards/vulnerability-disclosure.html.md.erb

```diff
@@ -31,7 +31,15 @@ You should use <https://vdp.cabinetoffice.gov.uk/.well-known/security.txt>
 in either:
 
 - the origin for your site's `/.well-known/security.txt`
-- the destination of a 302 redirect for `/.well-known/security.txt`
+- the destination of a redirect for `/.well-known/security.txt`
+
+A note on the redirect mechanism, try implementing in the following order to
+ensure the best capability with all user agents:
+
+1. Server-side redirect (302 status and `Location` header in response)
+2. Client-side HTML (meta `http-equiv=refresh` tag in the head)
+3. Client-side JavaScript redirect (`window.location.href`) - this won't work
+if JavaScript is disabled, so you should display a link as well
```

Comment:
I personally wouldn't line-break these, but not sure what the GDS Way is for this :upside_down_face: 

```suggestion
3. Client-side JavaScript redirect (`window.location.href`) - this won't work if JavaScript is disabled, so you should display a link as well
```

Reply:
Thanks! I personally like the line breaks for reviewing in terminal but I defo take on board that others will dislike or find the line breaks confusing. There's [some mention](https://gds-way.cloudapps.digital/manuals/programming-languages/python/python.html#maximum-line-length-of-120-characters) on max line length for _code_, but not markdown. I'll see where's best to flag this, perhaps I'll raise an issue for comments. I'm going to resolve these and merge from Tim's approval but will follow up on md line breaking :)